### PR TITLE
CEDS-1246 remove to metadata properties

### PR DESCRIPTION
**Pull request by PauloPortugal**
Also ran ScalaFmt on another commit

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -104,9 +104,9 @@ class AppConfig @Inject()(
     FeatureStatus.withName(loadConfig(feature2Key(Feature.default)))
 
   def wcoMetadataMapper(): WcoMetadataMapper with WcoMetadataMappingStrategy = {
-      logger.warn("Using WcoMetadataJavaMappingStrategy as the WCO-DEC mapper")
-      new WcoMetadataMapper with WcoMetadataJavaMappingStrategy
-    }
+    logger.warn("Using WcoMetadataJavaMappingStrategy as the WCO-DEC mapper")
```
Comment (PatrykRudnicki) [inline app/config/AppConfig.scala]: I think it should be debug, it's information for us, not warning. Old implementation, just commented.

File: app/config/AppConfig.scala [hunk 2]
```diff
@@ -103,10 +104,7 @@ class AppConfig @Inject()(
   lazy val defaultFeatureStatus: features.FeatureStatus.Value =
     FeatureStatus.withName(loadConfig(feature2Key(Feature.default)))
 
-  def wcoMetadataMapper(): WcoMetadataMapper with WcoMetadataMappingStrategy = {
-    logger.warn("Using WcoMetadataJavaMappingStrategy as the WCO-DEC mapper")
-    new WcoMetadataMapper with WcoMetadataJavaMappingStrategy
-  }
+  def wcoMetadataMapper(): WcoMetadataMapper = new WcoMetadataMapper
```
Comment (PatrykRudnicki) [inline app/config/AppConfig.scala]: Do you use this method somewhere? Is this method important? It just creates `WcoMetadataMapper`, so you can inline it.

File: app/config/AppConfig.scala (hunk 1 above)
Comment (PauloPortugal) [reply app/config/AppConfig.scala]: I have removed it ;)

Comment (PauloPortugal) [reply app/config/AppConfig.scala]: I have removed it

**Pull request merged**